test/e2e/testsuites/dynamically_provisioned_resize_volume_tester.go

```diff
@@ -82,5 +99,42 @@ func (t *DynamicallyProvisionedResizeVolumeTest) Run(client clientset.Interface,
 		if !newSize.Equal(updatedSize) {
 			framework.Failf("newSize(%+v) is not equal to updatedSize(%+v)", newSize, updatedSize)
 		}
+
+		ginkgo.By("checking the resizing PV result")
+		newPv, _ := client.CoreV1().PersistentVolumes().Get(context.Background(), updatedPvc.Spec.VolumeName, metav1.GetOptions{})
+		newPvSize := newPv.Spec.Capacity["storage"]
+		newPvSizeStr := newPvSize.String() + "Gi"
+
+		if !(newSize.String() == newPvSizeStr) {
+			framework.Logf("newPVCSize(%+v) is not equal to newPVSize(%+v)", newSize.String(), newPvSizeStr)
+		}
+
+		ginkgo.By("checking the resizing azurefile result")
+		var resourceGroup string
+		var accountName string
+		var shareName string
+		if newPv.Spec.PersistentVolumeSource.CSI != nil {
+			resourceGroup, accountName, shareName, err = GetSourceFromURI(newPv.Spec.PersistentVolumeSource.CSI.VolumeHandle)
+			framework.ExpectNoError(err, fmt.Sprintf("Error getting filesource for azurefile %v", err))
+		} else if newPv.Spec.PersistentVolumeSource.AzureFile != nil {
+			resourceGroup = ""
```

Comment:
I don't know how to get resourceGroup of azurefile here. What can I do for this? @andyzhangx

Reply:
use `creds.ResourceGroup` for e2e test